cm2350/tests/test_mpc5674_flash.py

```diff
@@ -789,12 +749,12 @@ def test_flash_erase_H0_interleaved(self):
         for addr in range(start, end, 32):
             # The first 16 bytes of each 32 byte chunk being compared should be
             # erased.
-            self.assertEqual(self.emu.readMemory(addr, 16), erased_chunk)
-            self.assertEqual(self.emu.flash.data[addr:addr+16], erased_chunk)
+            self.assertEqual(self.emu.readMemory(addr, 16), erased_chunk, msg=hex(addr))
+            self.assertEqual(self.emu.flash.data[addr:addr+16], erased_chunk, msg=hex(addr))
 
             # Second 16 bytes should not be erased since they belong to array B
-            self.assertEqual(self.emu.readMemory(addr+16, 16), unerased_chunk)
-            self.assertEqual(self.emu.flash.data[addr+16:addr+32], unerased_chunk)
+            self.assertEqual(self.emu.readMemory(addr+16, 16), unerased_chunk, msg=hex(addr+16))
+            self.assertEqual(self.emu.flash.data[addr+16:addr+32], unerased_chunk, msg=hex(addr+16))
```

Comment:
Is `hex(addr*)` enough of an error message?

Reply:
it should be, it should uniquely identify where the failure was.